app/src/main/res/layout/fragment_register_screen.xml

```diff
@@ -0,0 +1,314 @@
+                </com.google.android.material.textfield.TextInputLayout>
+
+                <com.google.android.material.textfield.TextInputLayout
+                    android:id="@+id/textInputLayout4"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:hint="@string/enter_last_name"
+                    app:boxStrokeColor="@color/Primary"
+                    app:hintAnimationEnabled="true"
+                    app:hintTextAppearance="@style/TextAppearance.AppCompat.Small"
+                    app:hintTextColor="@color/Disable"
+                    app:layout_constraintEnd_toEndOf="@id/txtLastName"
+                    app:layout_constraintStart_toStartOf="@+id/txtLastName"
+                    app:layout_constraintTop_toBottomOf="@+id/txtLastName">
+
+                    <com.google.android.material.textfield.TextInputEditText
+                        android:id="@+id/edt_last_name"
+                        style="@style/textInputEditTextStyle"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:inputType="textPersonName"
+                        tools:ignore="TextContrastCheck,SpeakableTextPresentCheck" />
+                </com.google.android.material.textfield.TextInputLayout>
+
+                <TextView
+                    android:id="@+id/txtEmailAddress"
+                    style="@style/fontBoldStyle"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:layout_marginEnd="24dp"
+                    android:layout_marginStart="24dp"
+                    android:text="@string/email_address"
+                    android:textSize="@dimen/fontHeading4"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/textInputLayout3"/>
+                <com.google.android.material.textfield.TextInputLayout
+                    android:id="@+id/textInputLayout"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:hint="@string/enter_email_address"
+                    app:boxStrokeColor="@color/Primary"
+                    app:hintAnimationEnabled="true"
+                    app:hintTextAppearance="@style/TextAppearance.AppCompat.Small"
+                    app:hintTextColor="@color/Disable"
+                    app:layout_constraintEnd_toEndOf="@id/txtEmailAddress"
+                    app:layout_constraintStart_toStartOf="@+id/txtEmailAddress"
+                    app:layout_constraintTop_toBottomOf="@+id/txtEmailAddress">
+
+                    <com.google.android.material.textfield.TextInputEditText
```

Comment:
it is beter to use name of fragment in id to know that tis view belongs to this fragment